Light.json

```diff
@@ -39788,7 +39822,8 @@
           "Warrior"
         ],
         "characteristics": [
-          "leader",
+          "guard",
+		  "leader",
```

Comment:
Spacing.

Reply:
Thanks! I have added "make sure I didn't use any tab characters" to my work process. There are also a few other tabs in there that I need to clean up, I will include those tab cleanups in my next update.